pallets/api/src/fungibles/mod.rs

```diff
@@ -93,6 +95,57 @@ pub mod pallet {
 	#[pallet::pallet]
 	pub struct Pallet<T>(_);
 
+	#[pallet::event]
+	#[pallet::generate_deposit(pub(super) fn deposit_event)]
+	pub enum Event<T: Config> {
+		/// Event emitted when allowance by `owner` to `spender` changes.
+		Approval {
+			/// Token ID.
+			id: AssetIdOf<T>,
+			/// Account providing allowance.
+			owner: AccountIdOf<T>,
+			/// Allowance beneficiary.
+			spender: AccountIdOf<T>,
+			/// New allowance amount.
+			value: BalanceOf<T>,
+		},
+		/// Event emitted when transfer of tokens occurs.
+		Transfer {
+			/// Token ID.
+			id: AssetIdOf<T>,
+			/// Transfer sender. `None` in case of minting new tokens.
+			from: Option<AccountIdOf<T>>,
+			/// Transfer recipient. `None` in case of burning tokens.
+			to: Option<AccountIdOf<T>>,
+			/// Amount of tokens transferred (or minted/burned).
+			value: BalanceOf<T>,
+		},
+		/// Event emitted when a token is created.
+		Create {
```

Comment:
This does not seem useful to me. It is simply emitting the same event that pallet-assets is already emitting

Reply:
I agree with you. Removed those events from the pallet.

Reply:
see refactor: remove duplicated events